libshortfin/CMakeLists.txt

```
@@ -23,21 +23,14 @@ set(CMAKE_CXX_STANDARD 20)
 # https://discourse.cmake.org/t/cmake-3-28-cmake-cxx-compiler-clang-scan-deps-notfound-not-found/9244/3
 set(CMAKE_CXX_SCAN_FOR_MODULES 0)
 
+# Problems with libfmt without fPIC
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Instead of modifying these CMAKE flags you probably want 
```suggestion
set (CMAKE_POSITION_INDEPENDENT_CODE ON)
```

---

oh that's cool! I'm not sure why this is happening though. My guess is it's some weird dependency web between spdlog and fmt.